client/src/app/site/base/base-filter.service/base-filter-list.service.ts

```diff
@@ -382,6 +382,32 @@ export abstract class BaseFilterListService<V extends BaseViewModel> implements
      */
     protected abstract getFilterDefinitions(): OsFilter<V>[];
 
+    private async loadFilters(): Promise<OsFilter<V>[] | null> {
+        return this.parseFilters(await this.activeFiltersStore.load<V>(this.storageKey));
+    }
+
+    private parseFilters(oldFilters: OsFilter<V>[]): OsFilter<V>[] {
+        const newFilterDefs = this.getFilterDefinitions();
+        newFilterDefs.forEach(definition => {
+            const oldDefinition = oldFilters.filter(old => old.property === definition.property);
+            if (!oldDefinition.length) {
+                return;
+            }
+            oldDefinition[0].options.forEach(option => {
+                if (typeof option === `string`) {
```

Comment:
why do you filter out strings?

Reply:
The point of this exercise is to get the old isActive values and parse them into the new filter settings from the code. If the option is just a string there is no isActive value to pull out, so they're ignored.

Reply:
Ah, I see. Is the possibility to provide just a string ever used anywhere? If not, maybe we should just remove it.

Reply:
I think it is not, I'll look into it

Reply:
There are indeed some places, where it is used, so I'll leave this in

Reply:
ok